tests/integration/portal-account.test.ts

```diff
@@ -0,0 +1,29 @@
+import { SkynetClient } from "skynet-js";
+
+import { randomAsciiString } from "../utils";
+import { generatePhrase, phraseToSeed } from "../../src/seed";
+import { getEmailFromJWT, login, register } from "../../src/portal-account";
+
+// TODO: Remove hard-coded URL.
```

Comment:
F/U: Act on this TODO. I'm only writing this, so we can capture it in a F/U issue.